### DISCUSSION TITLE
Update SPI without restarting Keycloak v22

### DISCUSSION THREAD

**Discussion by min-sung-kwon** [keycloak/keycloak#23719] [Q&A]
I am currently migrating from version 15 to version 22. 
In version 15, updates to the JAR file that implemented SPI were reflected without the need to restart the server, but in version 22, building and restarting were required.
Is there a way to update the SPI by simply updating the JAR file in the /providers directory without restarting the server?

**Comment by InterStelli** [comment]
Have you found a way around this?

**Comment by danielFesenmeyer** [comment]
Recently I learned that's not possible in the "legacy wildfly" way.
There seems to be another discussion around this topic, which at least shows a solution for Gradle: https://github.com/keycloak/keycloak/discussions/12595
